frame/evm/precompile/modexp/src/lib.rs

```diff
@@ -56,18 +57,18 @@ fn calculate_gas_cost(
 		words * words
 	}
 
-	fn calculate_iteration_count(exp_length: u64, exponent: &BigUint) -> u64 {
+	fn calculate_iteration_count(
+		exp_length: u64,
```

Comment:
Could this field be removed now since it should be redundant with `exponent_bytes.len()`?